src/en/css-shortcuts/box-sizing.md

```diff
@@ -0,0 +1,44 @@
+---
+title: CSS Shortcuts - Box sizing
+layout: 'layouts/base.njk'
+eleventyNavigation:
+  key: shortcutsBoxSizingEN
+  title: CSS Shortcuts - Box sizing
```

Comment:
Should we format the title and the eleventyNavigation like this for the CSS shortcuts file? That way the topic is read first in the page title and we don't have to remove `CSS shortcuts` from each menu item when we switch to showing them in the navigation.

```suggestion
title: Box sizing - CSS Shortcuts
layout: 'layouts/base.njk'
eleventyNavigation:
  key: shortcutsBoxSizingEN
  title: Box sizing
```

Reply:
Yes, thank you - good catch!